live/config-cdroot/fix_bootconfig.ppc64le

```diff
@@ -74,22 +79,22 @@ chmod 644 $boot_dir/{initrd,linux}
 cat >$dst/boot/grub2/grub.cfg <<XXX
 gfxmode=auto
 
-set default='Agama Live'
+set default="$kiwi_iname"
 
 insmod gettext
 
 if sleep --interruptible 0 ; then
   timeout=60
 fi
 
-menuentry 'Agama Live' --class os --unrestricted {
+menuentry "$kiwi_iname" --class os --unrestricted {
   echo 'Loading kernel...'
   linux /boot/ppc64le/linux
   echo 'Loading initrd...'
   initrd /boot/ppc64le/initrd
 }
 
-menuentry 'Check Installation Media -- Agama Live' --class os --unrestricted {
+menuentry "Check Installation Media -- $kiwi_iname" --class os --unrestricted {
```

Comment:
I felt that it would be nice to cosistently use $kiwi_iname across all arches. I did not dare to touch s390x.